controller/src/main/resources/org/jboss/as/controller/descriptions/common/LocalDescriptions.properties

```diff
@@ -531,6 +531,30 @@ global.query.where=a list of filter criteria tuples (i.e. 'running=true')
 global.query.select=a list of attribute names to reduce to
 global.query.operator=defines the semantics for the filter attributes (AND, OR)
 
+global.product-info=Get product info report
+global.product-info.file=The path to the file where the report is going to be saved.
+global.product-info.format=The format of the report [json|xml].
+global.product-info.summary=Summary of the current server installation.
+global.summary.report-version=Version of the summary of the current server installation.
+global.summary.host-cpu=The description of the host CPU.
+global.summary.host-cpu-arch=The CPU architecture of the host.
+global.summary.host-core-count=The number of cores available on the host.global.summary.java-home=The Java home.
+global.summary.host-name=The name to use for this host's host controller. Must be unique across the domain. If not set, defaults to the runtime value of InetAddress.getLocalHost().getHostName().
+global.summary.host-operating-system=The name of the Operating System.
+global.summary.instance-identifier=Unique Id of the server instance.
+global.summary.installation-date=Installation date of the current server instance.
+global.summary.java-home=Thepath to the JVM used by the server instance.
+global.summary.java-version=The Java language version.
+global.summary.jvm=The JVM configuration for the server.
+global.summary.jvm-version=The complete build version of the JVM.
+global.summary.jvm-vendor=The vendor of the JVM.
+global.summary.last-update-date=Date of the last update of the current product.
+global.summary.node-name=The name of the instance node.
+global.summary.product-community-identifier=Indicates the type of distribution, if it is a Product or a Project.
+global.summary.product-home=Installation directory of the current server instance.
+global.summary.product-name=The name of the WildFly based product that is being run by this server.
+global.summary.product-version=The version of the WildFly based product release that is being run by this server.
```

Comment:
Please remove "WildFly based". The info it adds isn't worth the possible hassle to remove it if someone wants it gone.